benchmarks/dynamo/common.py

```diff
@@ -1703,6 +1710,9 @@ def get_example_inputs(self):
         cause time measurement not accurate""",
     )
     parser.add_argument("--timing", action="store_true", help="Emits phase timing")
+    parser.add_argument(
+        "--recompiles", action="store_true", help="Emits recompile stats"
```

Comment:
Pick a different name, this is too close to verb `--recompile` and not at all clear what it does. `--print-recompiles` is an obvious alternate.